fs/write_json.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as path from "./path/mod.ts";
+
+interface WriteJsonOption {
+  spaces?: number | string;
+  replacer?: (number | string)[] | null | Replacer;
```

Comment:
This should be failing linting...  Also, why `null`?  That makes no sense.

```suggestion
  replacer?: Array<number | string> | Replacer;
```

Reply:
This is the definition of `JSON.stringify`

`replacer` can be null

```typescript
    /**
      * Converts a JavaScript value to a JavaScript Object Notation (JSON) string.
      * @param value A JavaScript value, usually an object or array, to be converted.
      * @param replacer A function that transforms the results.
      * @param space Adds indentation, white space, and line break characters to the return-value JSON text to make it easier to read.
      */
    stringify(value: any, replacer?: (key: string, value: any) => any, space?: string | number): string;
    /**
      * Converts a JavaScript value to a JavaScript Object Notation (JSON) string.
      * @param value A JavaScript value, usually an object or array, to be converted.
      * @param replacer An array of strings and numbers that acts as a approved list for selecting the object properties that will be stringified.
      * @param space Adds indentation, white space, and line break characters to the return-value JSON text to make it easier to read.
      */
    stringify(value: any, replacer?: (number | string)[] | null, space?: string | number): string;
```

Reply:
It _can_ be, but I wouldn't want people to use the API that way...  If we can tighten up APIs we should, even if what we are backing off to supports something different.

Reply:
Good point. I remove it.